README.md

```diff
@@ -399,6 +399,26 @@ to follow the [manual installation guide](#manual-installation) instead of using
 
 No, that's not necessary. The library will automatically complete and clean up every event stream for you when the component is destroyed.
 
+### What if I use typescript's strict mode?
+
+If you are working with TypeScript's strict mode or have set strictPropertyInitialization to true in your tsconfig.json, you will experience an error using @ObservableEvent().
```

Comment:
Use inline code blocks for `strictPropertyInitialization`, `tsconfig.json` and `@ObservableEvent()`